megamek/src/megamek/client/ui/swing/PlanetaryConditionsDialog.java

```diff
@@ -252,6 +255,16 @@ private void setupConditions() {
         c.anchor = GridBagConstraints.WEST;
         gridbag.setConstraints(choWind, c);
         panOptions.add(choWind);
+        
+        c.gridwidth = 1;
+        c.anchor = GridBagConstraints.EAST;
+        gridbag.setConstraints(labWindDirection, c);
+        panOptions.add(labWindDirection);
```

Comment:
Can you also add a call to setLabelFor? This will help with screen readers.

https://docs.oracle.com/javase/8/docs/api/javax/swing/JLabel.html#setLabelFor-java.awt.Component-

Reply:
I noticed a lot of copy and paste redundancy, so I addressed that as well.